### PR TITLE
Shrink docker image and bump version

### PR DESCRIPTION
**Pull request by davidr**
This PR bumps openzwave to the newest version and throws out a lot of stuff needed to produce ozwcp, but not to run it. This takes it down from 250MB or something like that to about 30MB.

I won't be offended if you don't merge this, but I figured I'd send you a PR.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM alpine:latest
 
-ENV OPENZWAVE_VERSION=510a37df958d7b7108781e7a16bfe52e5831514d
-ENV OPENZWAVE_CONTROL_PANEL_VERSION=bbbd461c5763faab4949b12da12901f2d6f00f48
+ENV OPENZWAVE_GIT_VERSION=1.4-2954
```
Comment (ruimarinho) [inline Dockerfile:3]: I don't see this variable being used. Can you remove this?

File: Dockerfile [hunk 2]
```diff
@@ -1,7 +1,8 @@
...
+ENV OPENZWAVE_VERSION=577868f
```
Comment (ruimarinho) [inline Dockerfile:4]: Can you do version bump on a different PR so that changes are segregated?

File: README.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ An OpenZWave docker image with the OpenZWave Control Panel built-in.
 
 ## Supported tags and respective `Dockerfile` links
 
-- `1.4.2784`, `1.4`, `latest` ([Dockerfile](https://github.com/ruimarinho/docker-openzwave/blob/master/Dockerfile))
+- `1.4.2954`, `1.4`, `latest` ([Dockerfile](https://github.com/ruimarinho/docker-openzwave/blob/master/Dockerfile))
```
Comment (ruimarinho) [inline README.md:10]: Seem comment above regarding version bump.

File: Dockerfile [hunk 4]
```diff
@@ -20,23 +21,28 @@ RUN apk --no-cache add \
...
     && git checkout ${OPENZWAVE_CONTROL_PANEL_VERSION} \
     && sed -i 's/#LIBUSB := -ludev/LIBUSB := -ludev/' Makefile \
     && sed -i 's/#LIBS := $(LIBZWAVE) $(GNUTLS) $(LIBMICROHTTPD) -pthread $(LIBUSB) -lresolv/LIBS := $(LIBZWAVE) $(GNUTLS) $(LIBMICROHTTPD) -pthread $(LIBUSB) -lresolv/' Makefile \
     && sed -i 's/LIBS := $(LIBZWAVE) $(GNUTLS) $(LIBMICROHTTPD) -pthread $(LIBUSB) $(ARCH) -lresolv/#LIBS := $(LIBZWAVE) $(GNUTLS) $(LIBMICROHTTPD) -pthread $(LIBUSB) $(ARCH) -lresolv/' Makefile \
     && make \
-    && ln -sd /root/open-zwave/config
+    && mv ozwcp cp.js cp.html /zwave \
+    && cd /zwave \
+    && rm -rf open-zwave open-zwave-control-panel \
+    && apk del .build-dependencies
```
Comment (ruimarinho) [inline Dockerfile:42]: Oops, I forgot to delete them indeed :)

File: Dockerfile [hunk 5]
```diff
@@ -20,23 +21,28 @@ RUN apk --no-cache add \
       linux-headers \
       make \
       openssl \
-    && cd /root \
+    && mkdir /zwave \
```
Comment (ruimarinho) [inline Dockerfile:24]: Is this directory change really necessary? It is a breaking change from the examples so those would have to be updated as well.

**Review by ruimarinho — CHANGES_REQUESTED** [review]

**Comment by ruimarinho** [comment]
@davidr thanks for the PR!

**Comment by davidr** [comment]
I'll close this one and reopen another couple
